packages/renderer/src/render.ts

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs';
+import * as path from 'path';
+import puppeteer from 'puppeteer';
+import {createServer} from 'vite';
+import {fileURLToPath} from 'url';
+
+const Root = fileURLToPath(new URL('.', import.meta.url));
+
+(async () => {
+  console.log('Rendering...');
+
+  const project = process.argv[2];
+  const metaPath = `${Root}/../../template/src/${project}.meta`;
+  if (!fs.existsSync(metaPath))
+    return console.log(`Project couldn't be found.`);
+
+  const [browser, server] = await Promise.all([
+    puppeteer.launch({headless: true}),
+    createServer({
+      root: Root,
+      configFile: path.resolve(Root, '../vite.config.ts'),
+      server: {
+        port: 9000,
```

Comment:
Should this port number be configurable?

Not exactly sure if this is related, but I noticed that running `npm run serve` offers the following convenience: will start on port `9000`, and if it's not available, use the next available e.g. `9001`.

Perhaps the port number here could follow a similar approach?

Reply:
Choosing the next available port is a Vite functionality. It will do it automatically.
Tho it may be a good idea to actually check what port Vite picked using:
```ts
server.httpServer.address().port
```
and redirect Puppeteer there.

Right now we're using a hardcoded `9000` which will work most of the time but will fail if something is already running on `9000`.